whatsmyname.go

```diff
@@ -16,11 +16,13 @@ func main () {
  	case "lucas":
  		fmt.Println("Handle These")
  	case "brandon":
+ 		fmt.Print("How many points are you ahead by?")
+ 		fmt.Scan(&points)
```

Comment:
that's the old version :-(



On Thu, May 12, 2016 at 1:00 PM, Ron Drabkin <notifications@github.com>
wrote:

> In whatsmyname.go
> <https://github.com/RonDrabkin/class_training_go/pull/1#discussion_r63086690>
> :
>
> > @@ -16,11 +16,13 @@ func main () {
> >   	case "lucas":
> >   		fmt.Println("Handle These")
> >   	case "brandon":
> > + 		fmt.Print("How many points are you ahead by?")
> > + 		fmt.Scan(&points)
>
> where did you declare the variable?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/RonDrabkin/class_training_go/pull/1/files/f80d57b3592788d95500d511ee9c434e2cf287a2#r63086690>
>